spec/ad_hoc_command_spec.rb

```diff
@@ -2,6 +2,7 @@
   let(:url)                 { "example.com/api/v1/ad_hoc_command_update/10" }
   let(:api)                 { AnsibleTowerClient::Api.new(instance_double("Faraday::Connection")) }
   let(:collection)          { api.ad_hoc_commands }
+  let(:described_class)     { api.ad_hoc_command_class }
```

Comment:
the default `described_class` won't always work.  Maybe I need to rethink the way I'm testing.